account_bank_statement_import/models/account_bank_statement_import.py

```diff
@@ -2,16 +2,19 @@
 """Framework for importing bank statement files."""
 import logging
 import base64
+from StringIO import StringIO
+from zipfile import ZipFile, BadZipfile  # BadZipFile in Python >= 3.2
 
 from openerp import api, models, fields
 from openerp.tools.translate import _
-from openerp.exceptions import Warning
+from openerp.exceptions import Warning as UserError
 
-_logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
 class AccountBankStatementLine(models.Model):
     """Extend model account.bank.statement.line."""
+    # pylint: disable=too-many-public-methods
```

Comment:
I like my pylint output to be as clean as possible. On the other hand this would be needed in every class based on models.Model, as the root problem lies in the Odoo code itself, which puts "everything but the kitchen sink" into one class.

I will remove it.